jaxrs-spec/src/main/asciidoc/chapters/environment/_javaee.adoc

```diff
@@ -197,8 +197,13 @@ The following additional requirements apply when using Managed Beans,
 CDI-style Beans or EJBs as resource classes, providers or `Application`
 subclasses:
 
-* Field and property injection of JAX-RS resources MUST be performed
-prior to the container invoking any `@PostConstruct` annotated method.
+* For JAX-RS resources where the JAX-RS implementation participates in
```

Comment:
Shouldn't this be "resources and providers"?

Reply:
Yes, I think that is more clear.  I'll add "and providers"